.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
```

Comment:
We probably also want a cargo doc check. I know this isn't a library, but checking for dead links and such is still nice.
See bevy_quickstart for the right flags.